src/util/segmentation/operations/eraseInsideRectangle.js

```diff
@@ -0,0 +1,28 @@
+import { eraseInsideBoundingBox } from './index';
```

Comment:
Why keep this function?

Reply:
`eraseInsideRectangle` is a strategy, `eraseInsideBoundingBox` is a potentially reusable function. The distinction of what are strategies here needs to be made clearer.